src/app/components/elements/panels/Banner.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode } from "react";
+import { Col, Container, Row } from "reactstrap";
+
+interface BannerProps {
+  id: string;
+  title: string;
+  subtitle: string;
+  link: string;
+  src: string;
+  alt?: string;
+  content: ReactNode;
```

Comment:
What you've done here is not wrong, but I wanted to mention that another alternative is to call this `children`, so that you can pass the content like this:
```typescript
<Banner>
  <div>my content</div>
</Banner>
```

You don't need to change this, but it's something to consider when creating new components. If it feels natural to do it this way, then you could set the `children` property.

Reply:
Yeah that makes sense, thanks. Also prompted me to read up a bit more on typing of react children!